examples/3d-tiles/tile-3d-layer.js

```diff
@@ -74,13 +71,56 @@ function commonSpacePlanesToWGS84(viewport) {
   }
 }
 
+function getFrameState(viewport, animationProps) {
```

Comment:
+1

Idea: This looks pretty generic. As a second step, we could perhaps move this calculation to Tileset3D. In that case we would want to extact the params from the context first before calling it, as we don't want Tileset3D to know about viewport and animationProps.